dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -1625,6 +1629,58 @@ private Workflow registerGatkSvWorkflow(WorkflowsApi ownerWorkflowApi) {
                 "test", "wdl", "/test.json");
         return ownerWorkflowApi.refresh(workflow.getId());
     }
+    
+    @Test
+    public void testWorkflowVersionAliasOperations() throws ApiException {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowApi = new WorkflowsApi(webClient);
+        workflowApi.manualRegister("github", "DockstoreTestUser2/dockstore_workflow_cnv",
+                "/workflow/cnv.cwl", "", "cwl", "/test.json");
+        final Workflow workflowByPathGithub = workflowApi.getWorkflowByPath(DOCKSTORE_TEST_USER2_RELATIVE_IMPORTS_WORKFLOW, null, false);
+        // do targetted refresh, should promote workflow to fully-fleshed out workflow
+        final Workflow workflow = workflowApi.refresh(workflowByPathGithub.getId());
+        workflowApi.publish(workflow.getId(), SwaggerUtility.createPublishRequest(true));
+
+        Assert.assertTrue(workflow.getWorkflowVersions().stream().anyMatch(versions -> "master".equals(versions.getName())));
+        Optional<WorkflowVersion> optionalWorkflowVersion = workflow.getWorkflowVersions().stream()
+                .filter(version -> "master".equalsIgnoreCase(version.getName())).findFirst();
+        assertTrue(optionalWorkflowVersion.isPresent());
+        WorkflowVersion workflowVersion = optionalWorkflowVersion.get();
+
+        // give the workflow version a few aliases
+        AliasesApi aliasesApi = new AliasesApi(webClient);
+        WorkflowVersion workflowVersionWithAliases = aliasesApi.addAliases(workflowVersion.getId(), "awesome workflowversion, spam, test workflowversion");
```

Comment:
It would be nice if you had variables for the alias names, especially the ones you test for -- eliminates the possibility of a typo.

Reply:
Left for another PR

Reply:
It would have taken literally 5 minutes or less with IDE refactoring to introduce a variable. If you create another issue and/or PR it will take a lot longer. Plus every duplicated string or code doubles the possibility of an error creeping in, which is why I'm anal about it.

But these are tests so I guess at this point so be it.

Reply:
Couldn't refactor and add variables because they are in comma separated quotes